submissions/maxim-zabolotny/html-movie-seat-booking-task/index.html

```diff
@@ -0,0 +1,446 @@
+<!DOCTYPE html>
```

Comment:
The design is too much similar to https://github.com/kottans/frontend-2021-classroom/pull/51
in case if you use his design parts, do you ask him about that?)

Reply:
I took the design from the site https://multiplex.ua/ru/

![image](https://user-images.githubusercontent.com/54500881/109416262-d12f7a80-79c5-11eb-8753-1d1cd6f331f2.png)

Reply:
Ok, no problem)